### DISCUSSION TITLE
Question about capacity value calculation process

### DISCUSSION THREAD

**Discussion by narata** [NREL/ReEDS-2.0#81] [Q&A: Features]
Hi, ReEDS team!

Very amazing of your huge job!
I am very interested in 7-year, per season & tech, top 10-hour capacity value calculation process, as function '**cc_vg()**' in **ReEDS_Augur/ E_capacity_credit.py**.
It seems to have a rigorous formula derivation, but I can not understand the physical meaning of related calculation, especially the line 446 '**load_ratio**' equation and the line 451 '**gen_tech**' equation. 
1. I don't know the physcial meaning of '**load_ratio**'.
2. I don't know the physical meaning of **two parts of 'gen_tech'** :
          gen_tech = vg_power[top_hours_net, :] + np.where(load_ratio < 1, vg_power[top_hours, :]*load_ratio, vg_power[top_hours, :]) 
3. What's more I wonder why '**load_ratio**' has relationship with '**gen_tech**'.

I've looked up previous technical reports and journal articals of yours, but no related drerivltion are found.
Could you give me some more details?
Many thanks!

Alex Song

**Comment by wesleyjcole** [comment]
Our method calculates the capacity credit for the fleet of variable renewable energy (VRE) technologies.  However, we want to assign capcaity credit to individual VRE techs.  This little bit of code is what does that.

You can imagine that during the peak hour there is a lot of wind and a lot of PV, but during the net peak hour this is little PV but still a lot of wind.  There are several ways you can determine this relative different in capacity credit contribution  We decided to weight the capacity credit by the relative amount of generation from each technology during the hours of load_dif and load_reduct.  We use combination of load_ratio and gen_tech to weight the relative contribution.

Happy to provide additional details if this doesn't make sense.

**Reply by narata** [reply]
Thanks for your response!  ​Now I can understand the function of the two line codes.

In augur, the CC of variable generations is calculated by the difference of LDC and NLDC, and then assigned to different technologies.
However, the top 10 hour time series of LDC and NLDC is different.  Can they subtract directly?
Second, I still could not understand the CC assignment rules among different variable generations. It seems  elusive for me...

Could you provide some references or additional dtails? Many thanks!

**Reply by wesleyjcole** [reply]
We have an internal slide deck that we put together to help our decision-making on how to do with allocation.  If you e-mail me at wesley.cole@nrel.gov I would be happy to share it with you along with some additional explanation.

**Reply by narata** [reply]
I have emailed you. Thanks!
